### DISCUSSION TITLE
User file/folder permissions for How to Install and Run WordPress on an EC2 Instance

### DISCUSSION THREAD

**Discussion by jont916** [CloudBytes-Academy/CloudBytes.dev#138] [Q&A]
After following [this ](https://cloudbytes.dev/aws-academy/how-to-install-and-run-wordpress-on-an-ec2-instance) guide to update a plugin you have to SFTP the file, but if you add `define('FS_METHOD', 'direct');` to wp-config.php then it will bypass the FTP, which is great for a development site, but you get a write error when trying to update a simple plugin. I believe the user is nginx and the group is www-data but the following commands does not seem to give write permission in Wordpress to simply update a plugin.

`sudo chown -R nginx:www-data /usr/share/nginx/html`

Wordpress Update Plugin Error Message: Could not create directory. /usr/share/nginx/html/wp-content/upgrade

**Comment by rehanhaider** [comment]
Have you found a solution yet?
